### PR TITLE
IDEM-206

### PR DESCRIPTION
**Pull request by JasminFarhi**

### CODE REVIEW

**Review by bitcoinbrisbane — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
+    "@types/react": "^18.0.15",
```
Comment (pondpiu) [inline package.json:36]: react is not need at root folder

**Review by pondpiu — CHANGES_REQUESTED** [review]
don't push package-lock.json when we're using yarn

**Comment by bitcoinbrisbane** [comment]
> don't push package-lock.json when we're using yarn

Good catch

File: app/app.json [hunk 2]
```diff
@@ -60,6 +60,12 @@
           "icon": "./assets/Bee-Logo.png",
           "color": "#ffffff"
         }
+      ],
+      [
+        "dependencies",
+        {
+          "@types/react": "^18.0.15"
```
Comment (amagon270) [inline app/app.json:67]: Why is this in app.json?  it should be in package.json under dev-dependencies

**Pull request closed without merging**